### PR TITLE
Improve bans rcon command pagination

### PR DESCRIPTION
**Pull request by Rei-Tw**
This improves readability and makes it easier to navigate through a large list of bans using pagination.

The list is empty :
image

The list contains few entries only :
image

Invalid page is sent to the rcon command :
image

Last page, containing less lines than the max entries per page :
image

PS: Note that during the tests banned IPs are randomized.

## Checklist

- [x] Tested the change ingame
- [x] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: src/engine/shared/netban.cpp [hunk 1]
```diff
@@ -479,13 +479,32 @@ void CNetBan::ConBans(IConsole::IResult *pResult, void *pUser)
...
+	const int End = Page * s_EntriesPerPage;
+	const int NumBans = pThis->m_BanAddrPool.Num() + pThis->m_BanRangePool.Num();
+	const int NumPages = NumBans / s_EntriesPerPage + 1;
 
-	int Count = 0;
 	char aBuf[256], aMsg[256];
+
+	if(NumBans == 0)
+	{
+		str_format(aMsg, sizeof(aMsg), "The ban list is currently empty.");
```
Comment (def-) [inline src/engine/shared/netban.cpp:493]:
```suggestion
		str_format(aMsg, sizeof(aMsg), "The ban list is empty.");
```

File: src/engine/shared/netban.cpp [hunk 2]
```diff
@@ -479,13 +479,32 @@ void CNetBan::ConBans(IConsole::IResult *pResult, void *pUser)
...
+		pThis->Console()->Print(IConsole::OUTPUT_LEVEL_STANDARD, "net_ban", aMsg);
+
+		return;
+	}
+
+	if(Page <= 0 || Page > NumPages)
+	{
+		str_format(aMsg, sizeof(aMsg), "Invalid page number. There %s currently %d %s available.", NumPages == 1 ? "is" : "are", NumPages, NumPages == 1 ? "page" : "pages");
```
Comment (def-) [inline src/engine/shared/netban.cpp:501]:
```suggestion
		str_format(aMsg, sizeof(aMsg), "Invalid page number. There %s %d %s available.", NumPages == 1 ? "is" : "are", NumPages, NumPages == 1 ? "page" : "pages");
```

**Review by def- — APPROVED** [review]

**Comment by heinrich5991** [comment]
Can you squash your commits?

**Comment by Rei-Tw** [comment]
> Can you squash your commits?

@heinrich5991 done !

**Review by Robyt3 — APPROVED** [review]

**Pull request merged**